entries/fadeIn.xml

```diff
@@ -40,7 +40,7 @@ $( "#clickme" ).click(function() {
       <img class="column three" src="/resources/0042_06_34.png" alt=""/>
       <img class="column three" src="/resources/0042_06_35.png" alt=""/>
       <img class="column three" src="/resources/0042_06_36.png" alt=""/>
-      <figcaption>figure 1</figcaption>
+      <figcaption>Figure 1 - Illustration of the <code>fadeIn()</code> effect</figcaption>
```

Comment:
+1 to @agcolom . There's an example of multiple images at http://www.w3.org/WAI/tutorials/images/groups/ too. It has nested figures/figcaptions, so the same might not apply in our case, but it suggests using `role="group"` and an `aria-labelledby` attribute. Your call.